configs/config.go

```diff
@@ -0,0 +1,125 @@
+package configs
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
+// PipelineConfig https://zhwt.github.io/yaml-to-go/
+type PipelineConfig struct {
+	SpecVersion    string         `yaml:"specVersion"`
+	Org            string         `yaml:"org"`
+	Kind           string         `yaml:"kind"`
+	Version        int            `yaml:"version"`
+	Name           string         `yaml:"name"`
+	Network        string         `yaml:"network"`
```

Comment:
can we add another variable "environment"? it will be for users to define it's "dev", "staging" or "production" or some other custom env

Reply:
we plan to use the same config file for both prod and staging..

for metadata db and destination db, if we specify `RDS_TEST_DB` or other names. these db config wil be overriden in our zrunner platform based on staging/prod

Reply:
done